hw/top_earlgrey/formal/conn_csvs/clkmgr_ast.csv

```diff
@@ -7,5 +7,10 @@
 
 ,NAME,SRC BLOCK,SRC SIGNAL,DEST BLOCK,DEST SIGNAL,,,,,,
 
-# clkmgr ast connectivity
+# ast -> clkmgr connectivity
+CONNECTION,AST_CLK0_CLKMGR,u_ast,clk_src_sys_o,top_earlgrey.u_clkmgr_aon,clk_main_i
+CONNECTION,AST_CLK1_CLKMGR,u_ast,clk_src_io_o,top_earlgrey.u_clkmgr_aon,clk_io_i
+CONNECTION,AST_CLK2_CLKMGR,u_ast,clk_src_usb_o,top_earlgrey.u_clkmgr_aon,clk_usb_i
+CONNECTION,AST_CLK3_CLKMGR,u_ast,clk_src_aon_o,top_earlgrey.u_clkmgr_aon,clk_aon_i
+# clkmgr -> ast connectivity
```

Comment:
Just a comment :)
I do not think the direction matters in connectivity check, but I think this will help review ;)